samples/matter/light_switch/Kconfig.sysbuild

```
@@ -4,31 +4,37 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+#### Radio core selection
 config NRF_DEFAULT_IPC_RADIO
 	default y
 
+# Enable IEEE802.15.4 serialization to network core
 config NETCORE_IPC_RADIO_IEEE802154
-	default y if BOARD_NRF5340DK_NRF5340_CPUAPP
+	default y if SOC_SERIES_NRF53X && !WIFI_NRF700X
 
+# Enable Bluetooth serialization to network core
 config NETCORE_IPC_RADIO_BT_HCI_IPC
-	default y if (BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP)
-
-config MATTER_FACTORY_DATA
-	default y if !BOARD_NRF21540DK
+	default y if SOC_SERIES_NRF53X
```

```suggestion
	default y if SOC_SERIES_NRF53X && WIFI_NRF700X
```
Shouldn't it be this way?